### PR TITLE
Jetpack Sync: Add additional sync args for theme actions for Activity Log

### PR DESCRIPTION
**Pull request by gititon**
Add additional sync args for theme actions for Activity Log

#### Changes proposed in this Pull Request:

Add additional sync args for theme actions for Activity Log

#### Testing instructions:

*phpunit

### CODE REVIEW

File: sync/class.jetpack-sync-module-themes.php [hunk 1]
```diff
@@ -65,8 +69,9 @@ public function sync_network_allowed_themes_change( $option, $value, $old_value,
 			 *
 			 * @param mixed $newly_disabled_themes, Array of info about network disabled themes
 			 * @param mixed $all_enabled_theme_slugs, Array of slugs of all enabled themes
+			 * @param bool $is_theme_deletion, Whether a theme was deleted Since 5.3
 			 */
-			do_action( 'jetpack_network_disabled_themes', $newly_disabled_themes, $all_enabled_theme_slugs );
+			do_action( 'jetpack_network_disabled_themes', $newly_disabled_themes, $all_enabled_theme_slugs, $is_theme_deletion );
```
Comment (lezama) [inline sync/class.jetpack-sync-module-themes.php]: when `is_theme_deletion ` should we send a `jetpack_network_deleted_themes` instead?

Comment (gititon) [inline sync/class.jetpack-sync-module-themes.php]: A theme deleted sync action is already being sent independently. This prevents the two activities from appearing in the activity log.

File: sync/class.jetpack-sync-module-themes.php [hunk 2]
```diff
@@ -414,9 +431,34 @@ private function get_theme_support_info() {
...
+		$theme_support['name'] = $theme->get('Name');
+		$theme_support['version'] =  $theme->get('Version');
+		$theme_support['slug'] = $theme->get_stylesheet();
+		$theme_support['uri'] = $theme->get('ThemeURI');
+
 
 		return $theme_support;
 	}
+
+	private function get_delete_theme_call() {
```
Comment (lezama) [inline sync/class.jetpack-sync-module-themes.php]: should we name this `is_theme_delete` and return a `bool` so both methods are similar?

Comment (gititon) [inline sync/class.jetpack-sync-module-themes.php]: that would have been my preference, but one of the values returned by that method is used https://github.com/Automattic/jetpack/pull/7555/files/0bc9c9158d7fbfc1172ad739e52979ff48a21184#diff-37af365e9198e952a002bbf543cdcdcfR151

**Comment by eliorivero** [comment]
cherry picked to 5.2 in f6ff9298d

**Pull request merged**